packages/protocol/contracts/thirdparty/lido/.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "lib/forge-std"]
+	path = lib/forge-std
+	url = https://github.com/foundry-rs/forge-std
```

Comment:
we have modules defined in `packages/protocol`, do not make another sub-foundry project inside an existing one.